README.md

```diff
@@ -155,3 +165,18 @@ or an array of numbers in case of a multi slider.
 ### License
 
 See the [License](LICENSE) file.
+
+### Developers
```

Comment:
We can remove this section for now and add it in another PR once we decide how development setup should work.